scrapers/covid-19/getTherapeutics.js

```diff
@@ -43,7 +43,7 @@ const getTherapeuticsData = async (keys, redis) => {
 			data: cleanData(parsedData)
 		}));
 	} catch (err) {
-		logger.err('Error: Requesting therapeutics data failed!', err);
+		logger.err('Error: Requesting therapeutics data failed!123', err);
```

Comment:
Oh I see, let's just change the message instead of adding numbers

Reply:
Oh crap, let me change that